code/modules/vehicles/mecha/mecha_wreckage.dm

```diff
@@ -0,0 +1,209 @@
+///////////////////////////////////
+////////  Mecha wreckage   ////////
+///////////////////////////////////
+
+
+/obj/structure/mecha_wreckage
```

Comment:
will mech wrechage even exist ? I don't think we have the sprites

Reply:
Kuro is redoing it already so ill see